functions/.gitignore

```diff
@@ -0,0 +1,9 @@
+# Compiled JavaScript files
```

Comment:
I think you should add `package-lock.json` in the gitignore file as well. This file should not be tracked by git as it is created just locally.